### ISSUE TITLE
Description/icon clash

### ISSUE THREAD

**Issue by robinpaulson** [martykan/forecastie#540]
![](https://i.imgur.com/oZEd7Vk.png)

See attached

**Comment by robinpaulson** [comment]
The phrase "Heavy intensity shower rain" is odd, it appears longer than necessary.

**Comment by robinpaulson** [comment]
"Heavy showers" sounds totally fine to me, less of a mouthful too.

**Comment by FridoDeluxe** [comment]
Int this case I'd rather fix the layout to handle potentail long weather descriptions than replacing this by another string. Going through all languages and finding appropriate shorter phrases doesn't sound like a good solution. There might always be a situation (small display size, wide system font ...) where the phrase gets too long and the layout needs to handles that.

Apart from that, if several of OWM's weather descriptions are odd we could think of a general mechanism to replace them by  more suitable phrases to enhance the user experience.

**Comment by robinpaulson** [comment]
> Going through all languages and finding appropriate shorter phrases doesn't sound like a good solution.

Yes, why would i think english is the only language with phrases which might overlap here?

> In this case I'd rather fix the layout to handle potential long weather descriptions

Yup

**Comment by FridoDeluxe** [comment]
> Apart from that, if several of OWM's weather descriptions are odd we could think of a general mechanism to replace them by more suitable phrases to enhance the user experience.

Some time ago I reported a wrong weather translation to the OWM support and suggested a better one, it got changed quite fast. In case weather descriptions are misleading or somehow cumbersome, there might be a chance to fix this directly at the source.

**Comment by robinpaulson** [comment]
@FridoDeluxe that sounds like a whole other issue, parallel to this

**Comment by robinpaulson** [comment]
Hmm, that last post of mine looks very dismissive in retrospect. I'm sorry about that @FridoDeluxe , my bad. I'll be more careful in future.

**Comment by FridoDeluxe** [comment]
> Hmm, that last post of mine looks very dismissive in retrospect. I'm sorry about that @FridoDeluxe , my bad. I'll be more careful in future.

Nothing to worry, doesn't sound dismissive to me. Everything's fine!
